From b3a25f8ce1203b88e7f7ede20ceed4b61f515fed Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 10 Jun 2022 15:22:37 +0800
Subject: [PATCH 146/274] MLK-25940-2 arm64: imx_v8_defconfig: build module for
 EXC3000 multi-touch

Default build the driver for EXC3000 multi-touch as module

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Reviewed-by: Jun Li <jun.li@nxp.com>
---
 arch/arm64/configs/imx_v8_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/imx_v8_defconfig b/arch/arm64/configs/imx_v8_defconfig
index dd593b24b..be98532e9 100644
--- a/arch/arm64/configs/imx_v8_defconfig
+++ b/arch/arm64/configs/imx_v8_defconfig
@@ -364,6 +364,7 @@ CONFIG_TOUCHSCREEN_ATMEL_MXT=m
 CONFIG_TOUCHSCREEN_GOODIX=m
 CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_TOUCHSCREEN_SYNAPTICS_DSX_I2C=m
+CONFIG_TOUCHSCREEN_EXC3000=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_PWM_VIBRA=m
 # CONFIG_SERIO_SERPORT is not set
-- 
2.25.1

